src/main/java/com/linkedin/kmf/partitioner/DefaultPartitioner.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2016 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.linkedin.kmf.partitioner;
+
+import static org.apache.kafka.common.utils.Utils.murmur2;
+
+
+public class DefaultPartitioner  implements Partitioner {
```

Comment:
Can you remove the extra space between `DefaultPartitioner` and `implements`? Also, we probably need both old producer partition and new producer partitioner r and it may be good idea to distinguish between them in the class name.